src/network/serverpackethandler.cpp

```diff
@@ -743,6 +746,9 @@ void Server::handleCommand_InventoryAction(NetworkPacket* pkt)
 		return;
 	}
 
+	// Player is active, so reset idle timer
+	player->resetIdletime( );
```

Comment:
(sorry for such nitpicky suggestions, but there should be no whitespace)
```suggestion
	player->resetIdletime();
```